### ISSUE TITLE
Typo in child modules docs

### ISSUE THREAD

**Issue by Rezyapkin-Vyacheslav** [PyTorchLightning/pytorch-lightning#2300]
## 📚 Documentation

_forward()_ doesn't return the decoder output. But in _training\_step()_ it used when calling _self(representation)_.  

    class Encoder(torch.nn.Module):
        pass

    class Decoder(torch.nn.Module):
        pass

    class AutoEncoder(LitMNIST):

        def __init__(self):
            super().__init__()
            self.encoder = Encoder()
            self.decoder = Decoder()

        def forward(self, x):
            generated = self.decoder(x)

        def training_step(self, batch, batch_idx):
            x, _ = batch

            representation = self.encoder(x)
            x_hat = self(representation)

            loss = MSE(x, x_hat)
            return loss

**Comment by paantya** [comment]
Not fix et(

@Rezyapkin-Vyacheslav 
please tell me, do you think it would be better to return self.encode (x) to forward?
I am somewhat confused as to why decoder (x) is returned

**Comment by Rezyapkin-Vyacheslav** [comment]
@paantya 
It depends on what way it is assumed to use forward function. I followed the usage case in training_step definition, where forward function is used as decoder.
